api/api.yaml

```diff
@@ -215,16 +215,27 @@ paths:
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
   /results/state:
+    description: >-
+      State API
     post:
-      tags: ['Metrics']
-      operationId: get_state_metrics
+      tags: ['State']
```

Comment:
State or States?

Reply:
The tag represents a grouping, hence 'States' (like 'Metrics') would be more appropriate.

Reply:
what about /results/state vs. /results/states ??   /results/metrics (plural) right now.

Reply:
you're right.  /results/states would go better.